src/azure-cli/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -793,8 +803,11 @@ def _flexible_server_params(command_group):
                 c.argument('public_access', options_list=['--public-access'], arg_type=get_enum_type(['Enabled', 'Disabled']),
                            help='Determines the public access. ')
 
-        with self.argument_context('{} flexible-server replica stop-replication'.format(command_group)) as c:
```

Comment:
as metioned earlier let's not remove stop-replication command completely. It is ok to remove it from help but don't remove support for the same.